app/services/diagnostics.ts

```diff
@@ -0,0 +1,637 @@
+/*global SLOBS_BUNDLE_ID*/
```

Comment:
Yup - webpack is compiling this variable into the bundle at compile time, and it's available in the global scope.  This comment tells eslint not to worry about it, since normally referencing undefined variables is something eslint flags.